### DISCUSSION TITLE
Getting "Turn" headlands to work ( modded greatplains YP-2425A )

### DISCUSSION THREAD

**Discussion by FuturisticMC** [Courseplay/courseplay#6828] [Q&A] [answered]
When I use the turn option for my corners on headlands when planting, as the tractor starts to back up it jack knifes. Im using the case magnum with a modded greatplains YP-2425A with a fertilizer tank that attaches solid on the back, it does not pivot

**Comment by Dwainepegg** [comment, marked as answer]
This happens to me also with the same planter. Turn option works great on Manure spreading, cultivating and plowing. I think it may be the planter causing the issue. Im going to try other seeder/planter options and let you know

Vanilla planter

**Reply by Dwainepegg** [reply]
This trailer has 2 options for attachment, The 3 Point hitch doesn't work. I changed it to Trailer hitch and it works fine

**Reply by FuturisticMC** [reply]
I think I chose the 3 point because i could turn sharper so that sucks

**Reply by pvaiko** [reply]
You may want to look at https://github.com/Courseplay/courseplay/issues/6823#issuecomment-778618009 , the implementWheelAlwaysOnGround setting may solve this problem.

**Reply by FuturisticMC** [reply]
> You may want to look at [#6823 (comment)](https://github.com/Courseplay/courseplay/issues/6823#issuecomment-778618009) , the implementWheelAlwaysOnGround setting may solve this problem.

so I have to put the xml in the modsettings folder then do I add a line for the mod in the mod section of the xml or do i add it where ever?

**Reply by pvaiko** [reply]
See the comments in this file for instructions: https://github.com/Courseplay/courseplay/blob/master/config/VehicleConfigurations.xml

**Reply by TinyCrocodile** [reply]
Great! 
Did not know about this for way to long!

**Reply by TinyCrocodile** [reply]
If i have a mod wich requires some of these settings do you want to add it to this file via pr of mine? 
Otherwise i keep it personal
I think it can get messy very quickly. 
But i am willing to share whatever i find.

**Reply by pvaiko** [reply]
Not sure what is the best way to do it, vanilla vehicles should definitely in that file, maybe very popular mods, for everything else the private config file would be better.

**Reply by Tensuko** [reply]
I think so too.
We should not add every mod in there, just some very popular mods, like KOECKERLING Vector 460 and KOECKERLING Vector 620 wich got requested to work with CP a few times.

**Reply by Tensuko** [reply]
It depends on what coupling you use. The 3-point one is bugged for the YP-2425A
